app/javascript/components/Greeting.js

```diff
@@ -0,0 +1,19 @@
+import React, { useContext, useEffect } from 'react';
+import { GreetingContext } from '../contexts/GreetingContext';
+
+function Greeting() {
+  const { state, dispatch } = useContext(GreetingContext);
+
+  useEffect(() => {
+    // Fetch the greeting from the Rails API
+    fetch('/random_greeting')
```

Comment:
- [ ] Your two apps are not successfully connecting together. Random greetings are not displayed. Let's kindly check screenshot below for more info
![Screenshot from 2023-10-30 18-16-44](https://github.com/rudicarrilloypr/hello-rails-react/assets/63087491/82e6ae92-2add-4467-8bd0-e187f504ba0c)